src/flows/pipes/Youtube/index.ts

```diff
@@ -0,0 +1,16 @@
+import View from './view'
+import './style.scss'
+
+export default register => {
+  register({
+    type: 'youtube',
+    family: 'test',
+    priority: -1,
+    featureFlag: 'flow-panel--youtube',
+    button: 'Youtube',
+    component: View,
+    initial: {
+      uri: 'nBHkIWAJitg',
```

Comment:
what is this random string?

Reply:
oh it's the youtube id of some influx video lol ok

Reply:
the id that follows a youtube video's url